Call_of_Cthulhu_7th_Ed/translations/sv.json

```diff
@@ -1,394 +1,394 @@
-	"zerog_ic-u": "Zero-G(05%)",
-	"talents-u": "Talents",
-	"shield-z": "Shield",
-	"head-u": "Head",
-	"body-u": "Body",
-	"l_arm-u": "L.Arm",
-	"chest-u": "Chest",
-	"r_arm-u": "R.Arm",
-	"l_leg-u": "L.Leg",
-	"abdmn-u": "Abdmn",
-	"r_leg-u": "R.Leg",
-	"item-u": "Item",
-	"defence-u": "Defence",
-	"weapons-u": "Weapons",
-	"combat-u": "Combat",
-	"weaponname-u": "Weapon",
-	"skill-u": "Skill",
-	"damage-u": "Damage",
-	"range-u": "Range",
-	"attacks-u": "Attacks",
+	"zerog_ic-u": "Noll-G(05%)",
+	"talents-u": "Talanger",
+	"shield-z": "Sköld",
+	"head-u": "Huvud",
+	"body-u": "Kropp",
+	"l_arm-u": "V.Arm",
+	"chest-u": "Bröstkorg",
+	"r_arm-u": "H.Arm",
+	"l_leg-u": "V.Ben",
+	"abdmn-u": "Mage",
+	"r_leg-u": "H.Ben",
+	"item-u": "Föremål",
+	"defence-u": "Skydd",
+	"weapons-u": "Vapen",
+	"combat-u": "Strid",
+	"weaponname-u": "Vapen",
+	"skill-u": "Färdighet",
+	"damage-u": "Skada",
+	"range-u": "Räckvidd",
+	"attacks-u": "Attacker",
 	"ammo-u": "Ammo",
-	"malf-u": "Malf",
-	"Fighting(Brawl)": "Fighting(Brawl)",
-	"Firarms(HG)": "Firarms(HG)",
-	"Firarms(SMG)": "Firarms(SMG)",
-	"Firarms(R/S)": "Firarms(R/S)",
-	"Fighting(Other)": "Fighting(Other)",
-	"Firarms(Electr.)": "Firarms(Electr.)",
-	"Throw": "Throw",
-	"Other Skill #1": "Other Skill #1",
-	"Other Skill #2": "Other Skill #2",
-	"Other Skill #3": "Other Skill #3",
-	"Other Skill #4": "Other Skill #4",
-	"Other Skill #5": "Other Skill #5",
-	"Other Skill #6": "Other Skill #6",
-	"Other Skill #7": "Other Skill #7",
-	"Other Skill #8": "Other Skill #8",
-	"Other Skill #9": "Other Skill #9",
-	"Other Skill #10": "Other Skill #10",
-	"spells-u": "Spells",
-	"companions&mounts-u": "Companions & Mounts",
-	"vehicles-u": "Vehicles",
-	"cap-u": "Capacity",
-	"vehicleattack-u": "Vehicular Impact Attack",
-	"operatorskill": "Operator's Skill",
-	"MountedWeaponSkill": "Mounted Weapon Skill",
-	"generalinfo-u": "General Info",
-	"cargo-u": "Cargo",
-	"passengers-u": "Passengers",
-	"san-u": "SAN",
-	"HP-z": "HP",
+	"malf-u": "Krångel",
+	"Fighting(Brawl)": "Strid(Handgemäng)",
+	"Firarms(HG)": "Skjutvapen(pistol)",
+	"Firarms(SMG)": "Skjutvapen(kulspruta)",
+	"Firarms(R/S)": "Skjutvapen(Gevär/hagelgevär)",
+	"Fighting(Other)": "Strid(Annat)",
+	"Firarms(Electr.)": "Skjutvapen(Elektr.)",
+	"Throw": "Kasta",
+	"Other Skill #1": "Annan färdighet #1",
+	"Other Skill #2": "Annan färdighet #2",
+	"Other Skill #3": "Annan färdighet #3",
+	"Other Skill #4": "Annan färdighet #4",
+	"Other Skill #5": "Annan färdighet #5",
+	"Other Skill #6": "Annan färdighet #6",
+	"Other Skill #7": "Annan färdighet #7",
+	"Other Skill #8": "Annan färdighet #8",
+	"Other Skill #9": "Annan färdighet #9",
+	"Other Skill #10": "Annan färdighet #10",
+	"spells-u": "Besvärjelser",
+	"companions&mounts-u": "Kumpaner & riddjur",
+	"vehicles-u": "Fordon",
+	"cap-u": "Kapacitet",
+	"vehicleattack-u": "Kollisionsattack",
+	"operatorskill": "Förarens färdighet",
+	"MountedWeaponSkill": "Monterad vapenfärdighet",
```

Comment:
This sounds a bit weird or archaic. Maybe "Beriden vapenfärdighet" or "Vapenfärdighet till häst" would work better?

Reply:
Thaaank you! Of course it's "Beriden vapenfärdighet". I was completely stuck thinking in terms of vehicles, looking for terminology regarding weapons mounted on them, that I didn't even stop to think about mounted as in riding...

Reply:
👍